src/test/java/com/segment/analytics/android/integrations/flurry/FlurryTest.java

```diff
@@ -149,6 +147,5 @@ public Builder create() {
     verifyStatic();
     FlurryAgent.setGender(Constants.FEMALE);
     verifyStatic();
```

Comment:
I think you can also remove this `verifyStatic()`, as it should be verifying invocation of the static method `setLocation`, which has been removed below.

Reply:
I don't understand this. I was assuming the removal of lines 121/122 was the way to go? How would I infer I also need to remove line 149?

Reply:
AH NEVERMIND